lib/jira/client/index.js

```diff
@@ -107,8 +107,8 @@ module.exports = async (id, installationId, jiraHost) => {
         delete: (repositoryId) => instance.delete(`/rest/devinfo/0.10/repository/${repositoryId}`, {
           fields: { _updateSequenceId: Date.now() }
         }),
-        update: (data) => instance.post('/rest/devinfo/0.10/bulk', {
-          preventTransitions: false,
+        update: (data, preventTransitions = false) => instance.post('/rest/devinfo/0.10/bulk', {
```

Comment:
What do you think about using a keyed value to improve clarity when calling the method?

```suggestion
        update: (data, { preventTransitions }) => instance.post('/rest/devinfo/0.10/bulk', {
```

The calling code is longer but might be worth the extra characters.

```javascript
jiraClient.devinfo.repository.update(jiraPayload, true)
// becomes
jiraClient.devinfo.repository.update(jiraPayload, { preventTransitions: true })
```

Reply:
Yeh definitely like the API better that way. Take a look at https://github.com/integrations/jira/pull/246/commits/4c1b6294cce3595bf59d9dcef1c2ace68343c659 for how I implemented it. Just throwing `{ preventTransitions }` doesn't work out because of how destructuring works, so instead I opted for just passing an options param.